md/en-us/about/about.md

```diff
@@ -10,15 +10,15 @@ g0v.tw is an online community that pushes information transparency, focusing on
 
 ### Origin of g0v
 
-In 2012, when the government released an advertisement of its Economic Power-Up Plan. Unsatisfied with government’s asymmetrical and non-transparent attitude, Wu became skeptical about its policy executing efficiency and direction. As a result, he and his friends together founded g0v.tw, and in December of the same year, the organization launched its first event, named “g0v.tw hackath0n” which attracted more than expected number of participants, and this online community officially started.
+In 2012, the government released a version of its Economic Power-Up Plan. Unsatisfied with government’s asymmetrical and non-transparent attitude, Wu became skeptical about its policy-executing efficiency and direction. As a result, he and his friends together founded g0v.tw. In December of that same year, the organization launched its first event, named “g0v.tw hackath0n” which attracted a more-than-expected number of participants, and this online community officially started.
```

Comment:
Mentioned "Wu" (ttcat?) seems a little out-of-context, as they are not mentioned elsewhere. Perhaps either link, use full name, or use nickname more commonly known outside community? :)